### ISSUE TITLE
How to get overall accuracy?

### ISSUE THREAD

**Issue by Sheetaldhar123** [AlexeyAB/darknet#2753]
How do I get overall accuracy on the entire training dataset. While testing, the accuracy is shown for individual images given to it. How to get overall test accuracy?

**Comment by AlexeyAB** [comment]
Use this command: https://github.com/AlexeyAB/darknet#when-should-i-stop-training

`./darknet detector map data/obj.data yolo-obj.cfg backup\yolo-obj_7000.weights`

**Comment by Sheetaldhar123** [comment]
Is map same as that of overall average accuracy?

**Comment by AlexeyAB** [comment]
@Sheetaldhar123 Yes: https://medium.com/@jonathan_hui/map-mean-average-precision-for-object-detection-45c121a31173
